packages/shopify-codemod/README.md

```diff
@@ -11,12 +11,32 @@ This repository contains a collection of Codemods written with [JSCodeshift](htt
 
 ## Included Transforms
 
+### `conditional-assign-to-if-statement`
+
+Changes conditional assignment of default values to if statements (see [`no-unused-expressions`.](http://eslint.org/docs/rules/no-unused-expressions))
+
+```sh
+jscodeshift -t shopify-codemod/transforms/conditional-assign-to-if-statement <file>
+```
+
+#### Example
+
+```js
+foo || (foo = 'bar');
+
+// BECOMES:
+
+if (!Boolean(foo)) {
```

Comment:
> a better negation

what does that mean

It's very clear that in JavaScript semantics `!` convert the RHS to a boolean, so I don't see any benefit in being explicit